core/change.go

```diff
@@ -100,85 +101,132 @@ func (change Change) Write(writer io.Writer) error {
 	return err
 }
 
-func kindFromLabel(config Config, label string) *KindConfig {
-	for _, kindConfig := range config.Kinds {
-		if kindConfig.Label == label {
-			return &kindConfig
-		}
+// AskPrompts will ask the user prompts based on the configuration
+// updating the change as prompts are answered.
+func AskPrompts(change *Change, config Config, stdinReader io.ReadCloser) error {
```

Comment:
Now that I think about it, passing in a change pointer to make changes to it feels a little odd, we should probably just have ask prompts be part of the change struct.

`func (c *Change) AskPrompts(...)`
Then I would just pass the config and stdin reader to each of the promptX methods.

Reply:
Yeah, I moved `AskPrompts` to the `Change` struct. I can remove the `Prompter` struct too.